irispy/spectrograph.py

```diff
@@ -229,17 +229,29 @@ def __init__(self, filenames, spectral_windows="All"):
 
     def __repr__(self):
         spectral_window = self.spectral_windows["name"][0]
-        spectral_windows_info = "".join(
-            ["\n    {0}\n        (raster axis: {1}, slit axis: {2}, spectral axis: {3})".format(
-            name, len(self.data[name].raster_axis), len(self.data[name].slit_axis),
```

Comment:
I believe that if there is only one element along the raster_axis, then ```self.data[name].raster_axis``` will be an array of length 1, not an integer.  If this is the case, then it should still work as written.